AlicerceTests/ServiceLocatorTests.swift

```diff
@@ -0,0 +1,695 @@
+//
+//  ServiceLocatorTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 26/01/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+class ServiceLocatorTests : XCTestCase {
+    
+    var serviceLocator = ServiceLocator()
```

Comment:
Sorry, i don't like implicit unwrapped so i avoid to use it all the time. but yeah, it makes sense.